.ci/package-testing/Jenkinsfile

```diff
@@ -0,0 +1,31 @@
+#!/bin/groovy
+library 'kibana-pipeline-library'
+kibanaLibrary.load()
+kibanaPipeline(timeoutMinutes: 300) {
+  slackNotifications.onFailure(disabled: true) {
```

Comment:
Wouldn't we want Slack notifications for failures?

Reply:
Pushed with 2c82a1b464f37c0be83634451e0784b80df292d4